### PR TITLE
Put the columns in all reports in a consistent order for #827

### PR DESCRIPTION
**Pull request by CJDennis**
- minor reformatting
- split some files during one test to prevent a PCRE error

### CODE REVIEW

File: src/Report/Html/Renderer/Template/coverage_bar.html.dist [hunk 1]
```diff
@@ -1,5 +1,7 @@
-       <div class="progress">
+
```
Comment (sebastianbergmann) [inline src/Report/Html/Renderer/Template/coverage_bar.html.dist:1]: Please avoid unnecessary (whitespace) changes.

Comment (CJDennis) [reply src/Report/Html/Renderer/Template/coverage_bar.html.dist:1]: Are tags supposed to line up? Because this one didn't.

Comment (sebastianbergmann) [reply src/Report/Html/Renderer/Template/coverage_bar.html.dist:1]: I do not know what you mean by that, sorry.

**Comment by sebastianbergmann** [comment]
This branch has conflicts that must be resolved.

File: src/Report/Html/Renderer/Template/coverage_bar.html.dist (hunk 1 above)
Comment (CJDennis) [reply src/Report/Html/Renderer/Template/coverage_bar.html.dist:1]: The generated source code was messy. It no longer is. Is indentation import to you? Either indent correctly, or have totally random indentation, or no indentation at all. Don't indent everything perfectly except for a couple of tags.

This is how it's currently formatted:
```
      <tr>
       <td class="danger">Total</td>
       <td class="danger big">       <div class="progress">
         <div class="progress-bar bg-danger" role="progressbar" aria-valuenow="0.00" aria-valuemin="0" aria-valuemax="100" style="width: 0.00%">
           <span class="sr-only">0.00% covered (danger)</span>
         </div>
       </div>
</td>
       <td class="danger small"><div align="right">0.00%</div></td>
       <td class="danger small"><div align="right">0&nbsp;/&nbsp;1</div></td>
       <td class="danger big">       <div class="progress">
         <div class="progress-bar bg-danger" role="progressbar" aria-valuenow="46.15" aria-valuemin="0" aria-valuemax="100" style="width: 46.15%">
           <span class="sr-only">46.15% covered (danger)</span>
         </div>
       </div>
</td>
```

**Pull request closed without merging**